docs/DeveloperGuide.adoc

```diff
@@ -729,8 +729,68 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
+
+|`* *` |user |edit my tags more conveniently |not need to delete all the tags when editing somebody's tags
+
 |`* *` |user |hide link:#private-contact-detail[private contact details] by default |minimize chance of someone else seeing them by accident
 
+|`* *` |basic user |be able to put tags of different colours on my contacts |identify them more easily
+
+|`* *` |new user |add more icons to the app |make it look prettier
+
+|`* *` |new user |see several interactive text fields like a template when I use the add command |not need to memorise the command for adding people
+
+|`* *` |user who uses other address book apps |import/export my address book to other format |transfer address book data between applications easily
+
+|`* *` |user who often contacts people |add Speed dial or a favourites list |easily contact those that I am in frequent touch with
+
+|`* *` |user with bad vision |adjust the font size of the address book |see the contents easily
+
+|`* *` |user who has contacts with many tags |have each tag with their own associated colour |customise my contacts better for my own visibility
+
+|`*` |user |look at a prettier address book |enhance my user experience
+
+|`*` |user |find people near me using their addresses |contact them more conveniently
+
+|`*` |user |record the birthdays of my contacts |know when to celebrate their birthdays
+
+|`*` |user |receive a notification on my contacts birthdays |know when they are and celebrate them
+
+|`*` |user |add chinese names |record my chinese friends with no english names
+
+|`*` |basic user |open the address book maximised |focus on the address book itself
+
+|`*` |advanced user |use shorter versions of a command |speed up finding contacts
+
+|`*` |new user |enter a confirmation before executing any delete command |make sure I do not delete the wrong person
+
+|`*` |user who likes nice fonts |change the address book font to something like quicksand |make it look prettier
+
+|`*` |user who likes variety |change the font colour based on my mood |feel better while using the address book
+
+|`*` |user who is concerned about security |use a login system |secure my contacts information
+
+|`*` |user with bad memory |receive a hint for correct commands when I type something similar that is incorrect |remove the hassle of memorising commands
+
+|`*` |user who frequently needs to send mail |create a mailing list |send mail more conveniently
+
+|`*` |user who loves convenience |access edit and delete functions quickly |use the address book more efficiently and save time
+
+|`*` |user who is also a programmer |search using regular expression |search for contacts more precisely
+
+|`*` |ultra-book user |adjust the application to my viewport |scale it down to the size of my laptop screen
+
+|`*` |user with many friends |see their birthdays collated and displayed in a calendar |know when to prepare for their birthdays
```

Comment:
Maybe this user story should be arranged closer to your other birthday-related user stories